spot_price_monitor/spot_price_monitor.py

```diff
@@ -38,14 +38,36 @@ def get_zones_from_k8s(k8s_client):
             )
     return list(availability_zones)
 
+def matches_label(label, node_labels):
+    ''' 
+    Returns a true if the label provided
+    is the node metadata labels. 
+    Supports both new and old schema
```

Comment:
Might be worth expanding on `new and old schema` because this comment is likely to have the context forgotten